terraform/deployments/mobile-backend/gha-iam-role.tf

```
@@ -0,0 +1,50 @@
+locals {
+  github_oidc_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:oidc-provider/token.actions.githubusercontent.com"
+}
+
+data "aws_iam_openid_connect_provider" "github_oidc" {
+  arn = local.github_oidc_arn
+}
+
+data "aws_iam_policy_document" "gha_image_attestation_role_permissions" {
```

Naming here might be misleading:
```suggestion
data "aws_iam_policy_document" "config_signing_role_permissions" {
```